.travis.yml

```diff
@@ -258,6 +258,12 @@ script:
   - opt_enable PRINTCOUNTER
   - build_marlin
   #
+  # Test REPORT_M408_JSON
+  #
+  - restore_configs
+  - opt_enable REPORT_M408_JSON
+  - build_marlin
+  #
```

Comment:
Enable the option in one of the other tests, rather than requiring an additional build.